cmd/argoexec/commands/root.go

```diff
@@ -76,11 +75,10 @@ func NewRootCommand() *cobra.Command {
 
 func initExecutor() *executor.WorkflowExecutor {
 	version := argo.GetVersion()
-	executorType := os.Getenv(common.EnvVarContainerRuntimeExecutor)
-	log.WithFields(log.Fields{"version": version.Version, "executorType": executorType}).Info("Starting Workflow Executor")
+	log.WithFields(log.Fields{"version": version.Version}).Info("Starting Workflow Executor")
 	config, err := clientConfig.ClientConfig()
 	checkErr(err)
-	config = restclient.AddUserAgent(config, fmt.Sprintf("argo-workflows/%s argo-executor/%s", version.Version, executorType))
+	config = restclient.AddUserAgent(config, fmt.Sprintf("argo-workflows/%s", version.Version))
```

Comment:
I think we can just remove the `executorType` but keep `argo-executor` to be consistent with UAs from controller, server, etc.

Reply:
hard code to emissary?

Reply:
Either `fmt.Sprintf("argo-workflows/%s emissary", version.Version)` or `fmt.Sprintf("argo-workflows/%s argo-executor", version.Version)` would work. The benefit of including a more general term like "executor" would give the operator a more meaningful signal when monitoring the traffic.

Reply:
`argo-executor=not-applicable`?

Reply:
So just `argo-executor` then?

Reply:
Sounds good